### PR TITLE
Maggie Hu_hw01

### PR DESCRIPTION
**Pull request by Maggie-106**
For the hw01, I added headers, links, emphasis, lists and a picture in my biology by editing the README.md. It was not that challenging but I still met some difficulties. 

I had some problems of locating the correct link of the picture. I solved it by creating an issue in my local repo and using that link. (I was also confused that why I couldn't create issues in the fork repo.) After talking with Donghyun, I realized that I could directly get the link by uploading the picture into the forked repo. Then I re-edited the README.md with the new link of the picture. 

I'm fairly fine with the Github workflow.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,29 @@
...
+  * Microeconomics
+  * Political Economy
+  * Econometrics
+
+* Programming course
+  * Computing for Social Sciences 
+  
+Here is a picture of me:
+
+![Maggie](https://github.com/Maggie-106/hw01/blob/master/Maggie.png?raw=true)
```
Comment (deblnia) [inline README.md:19]: Because this picture is in your repo, you can use a relative link like so: 
```
![Maggie](Maggie.png)
```

**Review by deblnia — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Improvement |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |       X    |              |            |
| **Coding strategy**         |           |       X      |            |
| **Presentation**            |    X       |              |            |
| **Achievement, creativity** |     X      |              |            |
| **Reproducibility**         |   X        |              |            |

Remarks:

* Overall, solid work! You've used more than four elements of markdown syntax, and it's good to hear that you're comfortable with the Git workflow (we'll be using it a lot 😄 ) 
* I'd be careful about how you link images -- the way you've done it does not effect presentation, but it does make you dependent on Github. If Github decides to change the format of their links, your README would break. Better to use a relative link.

File: README.md [hunk 2]
```diff
@@ -1 +1,29 @@
 # hw01
+
+## Biography 
+
+My name is **Maggie Hu** and I am a second-year graduate student at *Harris*.
+
+My courses include:
+
+* Professional courses
```
Comment (brendac29) [inline README.md:9]: Very clear and organized list.

**Comment by brendac29** [comment]
I like the way you used bullets to create an organized list of the classes you are taking. Your reflection is also clear in explaining the problem you encountered and how you went about resolving it. It is great that you included a picture of yourself so that we know who you are, but the image is very large compared to the rest of the elements in your file. I suggest [this resource](https://bookdown.org/yihui/rmarkdown-cookbook/figure-size.html) to help you adjust the size of images in the future. Furthermore, it would have been fun to use markdown syntax to include some personal information about yourself such as hobbies, interests, etc.

File: README.md [hunk 3]
```diff
@@ -1 +1,29 @@
 # hw01
+
+## Biography 
+
+My name is **Maggie Hu** and I am a second-year graduate student at *Harris*.
+
```
Comment (AdelineKao) [inline README.md:6]: I like how you emphasized your name as well as the school at the University of Chicago. However, I would recommend including the full name of the school (Harris School of Public Policy) for clarity.

File: README.md (hunk 2 above)
Comment (AdelineKao) [inline README.md:9]: I really liked how you organized you list and separated your "professional" courses from your "programming" course and further indented the courses listed under them.

File: README.md (hunk 1 above)
Comment (AdelineKao) [inline README.md:19]: I have no issues viewing the picture but I did notice that it was quite large. This is something I am also guilty of doing but I found that by adding {width = 30%} after the image path it should decrease the size of the image.

**Review by AdelineKao — COMMENTED** [review]
Overall, it was very well organized and easy to understand.

**Pull request closed without merging**